projects/storefrontlib/src/cms-components/product/current-product.service.ts

```diff
@@ -19,7 +20,7 @@ export class CurrentProductService {
   ) {}
 
   protected readonly PRODUCT_SCOPE =
-    this.features && this.features.isLevel('1.4') ? ['details'] : '';
+    this.features && this.features.isLevel('1.4') ? [ProductScope.DETAILS] : '';
```

Comment:
For consistency without wrapping in array

Reply:
👍